index.d.ts

```diff
@@ -74,6 +86,22 @@ declare module "replicate" {
     predictionId: string;
   }
 
+  export interface TrainingsCreateOptions {
+    destination: string;
+    version: string;
+    input: any;
+    webhook?: string;
+    webhook_events_filter?: WebhookEventType[];
+  }
```

Comment:
This doesn't match the options provided to `client.trainings.create` in the tests (`version` is absent). Is that intentional?